app/i18n/locales/en/homepage.json

```diff
@@ -1,6 +1,6 @@
 {
   "headerTitle": "Privacy + Scaling Explorations",
-  "headerSubtitle": "Building free resources for people expanding the world of programmable cryptography.",
+  "headerSubtitle": "PSE is a research and development lab with a mission of making cryptography useful for human collaboration. We build open source tooling with things like zero-knowledge proofs, multiparty computation, homomorphic encryption, Ethereum, and more.",
```

Comment:
Here is the new description for the homepage
it needs changes? cc:  @ETHorHIL @samajammin